tests/RulesetEngineTest.php

```diff
@@ -37,7 +36,6 @@ public function dependencyProvider(): iterable
                 'LayerA' => [
                     'LayerB',
                 ],
-                'LayerC' => [],
```

Comment:
this line should be fine?

Reply:
This array describes the `$rulesetConfiguration` and there is no reference in the arrays before.